cdap-docs/integrations/source/partners/cloudera/configuring.rst

```diff
@@ -85,11 +85,25 @@ Details on CSDs and Cloudera Manager Extensions are `available online
 Install, Setup, and Startup
 =======================================
 
-.. rubric:: Install the CSD
+.. _cloudera-configuring-csd-parcel:
+
+.. rubric:: Install the CSD, Download and Distribute Parcel
 
 #. `Install the CSD <http://www.cloudera.com/content/cloudera/en/documentation/core/latest/topics/cm_mc_addon_services.html>`__.
-#. Download and distribute the CDAP-|version| parcel. If the Cask parcel repo is
-   inaccessible to your cluster, please see :ref:`these suggestions <cloudera-direct-parcel-access>`.
+#. Download and distribute the CDAP-|version| parcel. Complete instructions on parcels are available at
+   `Cloudera's website <http://www.cloudera.com/content/cloudera/en/documentation/core/latest/topics/cm_ig_parcels.html>`__,
+   but in summary there are four steps:
+   
+   1. `Adding the repository <http://www.cloudera.com/content/cloudera/en/documentation/core/latest/topics/cm_ig_parcels.html#cmug_topic_7_11_5_unique_1>`__
+      to the list of repositories searched by Cloudera Manager;
```

Comment:
Maybe just mention that installing the CSD adds the corresponding CDAP repository for you, but you can still customize it if you want.  (for example, CSD-3.1.x will point to the CDAP 3.1 parcel repo where CDAP 3.1.x parcels will be available.)